includes/class-wcpdf-main.php

```diff
@@ -341,6 +348,24 @@ public function generate_pdf_ajax() {
 			if ( $document ) {
 				do_action( 'wpo_wcpdf_document_created_manually', $document, $order_ids ); // note that $order_ids is filtered and may not be the same as the order IDs used for the document (which can be fetched from the document object itself)
 
+				// log document creation to order notes
+				$debug_settings = get_option( 'wpo_wcpdf_settings_debug', array() );
+				if( isset( $debug_settings['log_to_order_notes'] ) ) {
+					if( count( $order_ids ) > 1 ) {
+						$action = __( 'bulk', 'woocommerce-pdf-invoices-packing-slips' );
+						foreach( $order_ids as $order_id ) {
```

Comment:
I would do this looping inside the log function itself (checking `$document->is_bulk`?)